### PR TITLE
ESLint: Clean up unnecessary shared linting rules, add automated require sorting

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

File: shared/src/business/entities/Case.js [hunk 1]
```diff
@@ -14,6 +14,8 @@ const uuidVersions = {
   version: ['uuidv4'],
 };
 
+// ====;=;=;=;=;=;=\\\/\/\/\//\/\\
```
Comment (rachaelparris) [inline shared/src/business/entities/Case.js]: What's this?

**Review by wbyoko — APPROVED** [review]

File: shared/src/business/entities/Case.js (hunk 1 above)
Comment (ericsorenson) [reply shared/src/business/entities/Case.js]: a mistake 🥴

Comment (rachaelparris) [reply shared/src/business/entities/Case.js]: I was wondering if something automated created this, and why. Ha!

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**